cmd/containerd/main.go

```diff
@@ -1,9 +1,11 @@
 package main
 
 import (
+	"expvar"
 	"fmt"
 	"net"
 	"net/http"
+	"net/http/pprof"
```

Comment:
`_ "net/http/pprof`

and you can remove the handler you're registering, the init of the module does it already.